controldev.orogen

```diff
@@ -84,3 +84,14 @@ task_context "Remote", subclasses: 'GenericTask' do
     port_driven
 end
 
+task_context "RawCommandAggregator" do
+    input_port("command_in", "controldev/RawCommand")
+        .multiplexes
+        .needs_reliable_connection
```

Comment:
It's unlikely that you need a reliable connection.... Does the component need a guarantee that all samples arrive from its inputs ?

Reply:
Not at all, actually it is the exact opposite. Tks